### PR TITLE
chore: add structured package data for `math/base/special/cbrt`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   adds structured package data for [`math/base/special/cbrt`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/cbrt).

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves a part of #1147.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/special/cbrt/package.json [hunk 1]
```diff
@@ -64,5 +64,81 @@
...
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "cbrt",
+      "alias": "cbrt",
+      "pkg_desc": "compute the cube root",
+      "desc": "computes the cube root",
```
Comment (gunjjoshi) [inline lib/node_modules/@stdlib/math/base/special/cbrt/package.json:74]: In these descriptions, should we mention whether the package is for single-precision or double-precision numbers?

Comment (gunjjoshi) [reply lib/node_modules/@stdlib/math/base/special/cbrt/package.json:74]: Something like, `compute/computes the cube root of a single/double-precision floating-point number`.

Comment (kgryte) [reply lib/node_modules/@stdlib/math/base/special/cbrt/package.json:74]: No. I think we can rely on scaffolding to insert that language where appropriate. Here, we only want the "essential" bit of the description which distinguishes this package from others.

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @gunjjoshi!

**Pull request merged**